Week1/Js-exercises/8-type-checker.js

```diff
@@ -0,0 +1,62 @@
+
+"use strict";
+let var1 = "customers";
+let var2 = "suppliers";
+let var3 = {storeName:"mainStore",storeId:001};
+let var4 = {itemName:"laptop",itemId:0200};
+
+
+console.log(typeof var1);
```

Comment:
👍 for implementation but you could use `const` instead of `let` here

Reply:
Fixed